### DISCUSSION TITLE
[灌水] 提醒各位喜欢去404live的市民

### DISCUSSION THREAD

**Discussion by ZhuYuan-148** [share121/inter-knot#991] [其他]
![IMG_20240810_201209](https://github.com/user-attachments/assets/3d3466fb-7b9d-4f77-a3c6-9e210938b02f)
今天七夕节，被random play的店长带着去听了404live，现场氛围很好，我对其中一个乐队很感兴趣，好像叫什么，“白沙发”？ 现场有小扒手趁乱偷包，还好是被我注意到了，实施了抓捕，只不过很多人以为我是在cos罢了…

还请各位市民注意安全，新艾利都治安局会永远保护你们
![1723215847706](https://github.com/user-attachments/assets/90c56c7a-9995-484c-80fd-0d0c59457f50)

**Comment by Yizakl** [comment]
为什么在哪儿都能看到您？您不休息吗

**Comment by gaojiaobei** [comment]
朱鸢酱，我是你的粉丝啊！（飞扑）
